materials/where_to_go.md

```diff
@@ -10,34 +10,26 @@ Computing course project available until 20.10.23
 
 ## Connecting to the supercomputer via SSH
 
-We have accessed the supercomputer via the webinterface in order to not overwhelm you with setups before the course. However, this way may not always be the most convenient. If you want to access the supercomputer from your terminal directly, you will have to set up SSH keys:
+We have accessed the supercomputer via the webinterface in order to not overwhelm you with setups before the course. However, this way may not always be the most convenient. Direct SSH connection likely works better.
 
-:::{admonition} SSH keys and setup
+:::{admonition} Connecting with SSH clients
 :class: seealso, dropdown
 
-**Login with SSH**
-
-- SSH is a terminal program that gives you command-line access on the CSC supercomputer
-- It is a versatile main interface to a supercomputer
-   - Laptop &harr; Toyota, Supercomputer &harr; F1. F1 needs a specialist interface.
-- Please read this page for an introduction on [how to log in with SSH](https://docs.csc.fi/computing/connecting/)
-   - Mac and Linux have SSH. On Windows, Powershell can be used, but we recommend the Puhti web interface, or clients like MobaXterm or PuTTY
-   - Note the [prerequisites to be able to access Puhti](https://docs.csc.fi/support/faq/how-to-get-puhti-access/)
-- Plain SSH will not allow displaying remote graphics
-   - Puhti web interface is often best for this, but can be enabled also by X11-tunneling (additional installations required on Windows, see link above)
-
-**Setting up SSH keys**
-
-- Using SSH keys is easier and safer than using a password with every login
-- SSH keys can be easily used in Windows, Mac, Linux
-- Consult our [tutorials on how to set up SSH keys for your account](https://docs.csc.fi/computing/connecting/#setting-up-ssh-keys)
-   - [Logging in to LUMI](https://docs.lumi-supercomputer.eu/firststeps/getstarted/) requires setting up an SSH key pair and registering the public key in [My CSC](https:/my.csc.fi)
+- SSH clients give command-line access to a supercomputer or any other Linux server.
+- Basic SSH connection will not allow displaying remote graphics, but it is possible to set up with extra settings, but in case of Puhti web interface is recommended.
+- SSH clients:
+   - Mac and Linux have Terminal for SSH connections.
+   - In Windows:
+      - `Command Prompt` or `Powershell` are always avaialbe and can be used for basic connections.
+      - Special tools like [PuTTY](https://www.putty.org/) or [MobaXterm](https://mobaxterm.mobatek.net/) provide more options, inc possibility to save settings, but need installation.
+- [CSC Docs: Connecting to CSC supercomputers](https://docs.csc.fi/computing/connecting/)
+- [LUMI Docs: Get started](https://docs.lumi-supercomputer.eu/firststeps/).
```

Comment:
If you want to avoid typing your password every time you log in to the supercomputer, you can also set up ssh-keys, which in addition also make your connection more secure. You can find instructions for different tools in [CSC Docs: Setting up ssh-keys](https://docs.csc.fi/computing/connecting/#setting-up-ssh-keys).